src/twisted/application/twist/_twist.py

```diff
@@ -102,8 +102,12 @@ def main(cls, argv: Sequence[str] = sys.argv) -> None:
         options = cls.options(argv)
 
         reactor = options["reactor"]
+        # If subCommand is None, TwistOptions.parseOptions() raises UsageError
+        # and Twist.options() will exit the runner, so we'll never get here.
+        subCommand = options.subCommand
+        assert subCommand is not None
```

Comment:
I think that this needs an inline comment to note that this line was added to make mypy happy... as if you remove it, mypy wil fail.

```suggestion
        assert subCommand is not None  # Here to help mypy.
```

Reply:
I think the comment is covered in 105-106